연습문제/요격 시스템/PGS_요격시스템_이지영.java

```diff
@@ -0,0 +1,53 @@
+import java.util.*;
+
+public class PGS_요격시스템_이지영 {
+
+    class Target implements Comparable<Target> {
+        int start;
+        int end;
+
+        public Target(int start, int end) {
+            this.start = start;
+            this.end = end;
+        }
+
+        public int compareTo(Target o) {
+            if (this.end == o.end) {
+                if (this.start < o.start) return -1;
+                else return 1;
+            }
+            if (this.end > o.end) return -1;
+            else return 1;
```

Comment:
```suggestion
            if (this.end == o.end) {
                return Integer.compare(this.start,o.start);
            }
            return Integer.compare(o.end,this.end);
```
Integer.compare() 메서드를 이용해서 간단하게 표현 가능할 것 같아요!